js/functional/palindrome.js

```diff
@@ -1,13 +1,25 @@
 import { parseString } from "../helper.js";
 
+/**
+ * Add cleanString to avoid repeating the cleaning process for cleanedValue in each function.
+ *
+ * @param {string} value
+ */
+function cleanString(value) {
+  return value.toLowerCase().replace(/[^a-z0-9]/g, "");
+}
+
 /**
  * Determine whether the given value is a palindrome or not using reverse way.
  *
  * @param {string} value
  */
 function isPalindromeUsingReverse(value) {
-  // write your code here
-  return true;
+  value = parseString(value);
+
+  const cleanedValue = cleanString(value);
+  const reversedValue = cleanedValue.split("").reverse().join("");
+  return cleanedValue === reversedValue;
```

Comment:
Put an extra line break at this "return" statement to make it easier to read by the other.